packages/shared/src/components/ShareBar.tsx

```diff
@@ -56,7 +56,7 @@ export default function ShareBar({ post }: { post: Post }): ReactElement {
             target="_blank"
             rel="noopener"
             onClick={() => onClick('whatsapp')}
-            icon={<WhatsappIcon />}
+            icon={<WhatsappIcon style={{ fontSize: '1.75rem' }} />}
```

Comment:
Can't we set a size for this as well?

Reply:
This icon is not a component, but a simple .svg and I think we can keep it as it is